app/code/community/Op/Checkout/Model/Api/Checkout.php

```diff
@@ -335,8 +335,9 @@ public function _itemArgs($order)
         }
 
         if (!$order->getIsVirtual()) {
+
             $shippingExclTax = $order->getShippingAmount();
-            $shippingInclTax = $order->getShippingInclTax();
+            $shippingInclTax = $order->getShippingAmount() + $order->getBaseShippingTaxAmount();
```

Comment:
hm... base ? base is in base currency. should this be getShippingTaxAmount ?

Reply:
Yes, I was actually just changing it

Reply:
Nice.
Test, commit and I will approve then